akka-docs/src/main/paradox/stream/operators/flow/lazyInitAsync.md

```diff
@@ -0,0 +1,40 @@
+# Flow.lazyInitAsync
+
+Creates a real `Flow` upon receiving the first element by calling relevant `flowFactory` given as an argument.
+
+@ref[Simple processing stages](../index.md#simple-processing-stages)
+
+@@@div { .group-scala }
+
+## Signature
+
+@@signature [Flow.scala]($akka$/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #lazyInitAsync }
+
+@@@
```

Comment:
let's make a ticket for making `@@signature` that can do `@@javasignature`

Reply:
:+1: #24931